Storage/Storage/Model/SitePlugin+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import CoreData
+
+
+extension SitePlugin {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<SitePlugin> {
+        return NSFetchRequest<SitePlugin>(entityName: "SitePlugin")
+    }
+
+    @NSManaged public var siteID: Int64
+    @NSManaged public var plugin: String?
+    @NSManaged public var status: String?
+    @NSManaged public var name: String?
+    @NSManaged public var pluginUri: String?
+    @NSManaged public var author: String?
+    @NSManaged public var authorUri: String?
+    @NSManaged public var descriptionRaw: String?
+    @NSManaged public var descriptionRendered: String?
+    @NSManaged public var version: String?
+    @NSManaged public var networkOnly: Bool
+    @NSManaged public var requiresWPVersion: String?
+    @NSManaged public var requiresPHPVersion: String?
+    @NSManaged public var textDomain: String?
```

Comment:
Good catch! I usually keep the variables as-is to avoid having to change them again when there is any update in the future. But since it is less likely for changes to happen in this case (the API seems stable), I think it's a good idea to make the variables non-optional and keep code clearer when working with them later on.